batch-index/src/main/java/no/unit/nva/search/S3IonReader.java

```diff
@@ -0,0 +1,57 @@
+package no.unit.nva.search;
```

Comment:
This class was transfered from publication-api and modified for better memory usage. The next step will be to be transfered to commons and used by publication-api as well.

Reply:
It is tested indirectly, through the tests of the batch indexer